content/tools/polyfill.md

```diff
@@ -0,0 +1,80 @@
+---
+title: HTML & CSS Polyfills
+sub: Cascade Layers, CSS Anchor Positioning, Popover
+feature: true
+date: 2022-02-11
+image:
+  position: center
+  src: projects/polyfill.jpg
+oss: owner
+author: oddbird
+action:
+  text: Explore Polyfills »
+  url: https://github.com/orgs/oddbird/repositories?q=polyfill&type=all&language=&sort=
+index: Polyfill
+links:
+  source: https://github.com/orgs/oddbird/repositories?q=polyfill&type=all&language=&sort=
+summary: |
+  Along with our work for the W3C
+  developing specifications
+  for the CSS language,
+  OddBird has started and maintains
+  a number of powerful polyfills
+  for new web platform features
+  including Cascade Layers, CSS Anchor Positioning,
+  and the Popover attribute.
+---
+
+{% import 'embed.macros.njk' as embed %}
+
+## Cascade Layers Polyfill
+
+[Demo](https://layers-polyfill-example.netlify.app/)
+[Source](https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-cascade-layers#readme)
+
+PostCSS Cascade Layers lets you use `@layer`
+following the Cascade Layers Specification.
+For more information on layers,
+checkout [A Complete Guide to CSS Cascade Layers](https://css-tricks.com/css-cascade-layers/)
+by Miriam Suzanne.
+
+## CSS Anchor Positioning Polyfill
```

Comment:
We could add in a screenshot of https://anchor-position-wpt.netlify.app/ here to break up the text?

Reply:
Or maybe a codepen demo embed?

Reply:
@mirisuzanne Do you have a codepen demo or can you point me to one that you'd like to include?

Reply:
@jgerigmeyer I don't know what I'm looking at here. Is this what you would expect to see if we included a screenshot?
<img width="1440" alt="Screenshot 2023-11-07 at 2 30 10 PM" src="https://github.com/oddbird/oddleventy/assets/8933115/9aac0f64-8111-4340-828c-59cc2ff5299a">

Reply:
I don't know that we have a codepen demo, but we have a polyfill and a demo page - so I'm betting we could make one. And I think it would be much more clear to understand what we're showing, compared to the green and (mostly) red boxes of the WPT panel.

Reply:
Yeah, a codepen or screenshot of https://anchor-polyfill.netlify.app/ makes sense to me.

Reply:
@mirisuzanne 
**Options**:
a. Would you like me to wait to make this update live until after you add a codepen?
b. Would you like me to use my screenshot, go live with this page, and create a separate card for the codepen?
c. Would you like me to _not_ use any screenshot, go live with this page, and create a separate card for the codepen?
d. Or some other option?

Reply:
@jgerigmeyer Do you have a leaning here, Jonny? I think Mia is super busy this week. If I can publish this to oddsite, then I can go ahead start talking about the opportunity to sponsor work like this on the socials.

Reply:
A screenshot with a link to the demo seems like a fine first step? No need to block on having an embed.